ruby-training/trainees/alan/test_katas/02/fizzbuzz/fizzbuzz_game.rb

```diff
@@ -1,31 +1,31 @@
 class FizzBuzzGame
     
-    def self.it_is_fizz? (number)
+    def self.is_it_fizz? (number)
         number % 3 == 0
     end
 
-    def self.it_is_buzz? (number)
+    def self.is_it_buzz? (number)
         number % 5 == 0
     end
 
-    def self.it_is_fizzbuzz? (number)
-        it_is_buzz?(number) and it_is_fizz?(number)
+    def self.is_it_fizzbuzz? (number)
+        is_it_buzz?(number) and is_it_fizz?(number)
     end
 
-    def self.it_has_a_3? (number)
+    def self.has_it_a_3? (number)
         number.to_s.include?("3")
     end
 
-    def self.it_has_a_5? (number)
+    def self.has_it_a_5? (number)
```

Comment:
This class is over-fragmented in its methods. I think `contains_a_five?` should either be called inside `is_it_buzz?` or the contents of the method themselves should be put inside `is_it_buzz` (same for the "3" and "fizz" case). Having too many single-line methods can also make the code start to "smell".